Microsoft.Toolkit.Uwp.UI.Controls/SlidableListItem/SlidableListItem.cs

```diff
@@ -245,6 +249,14 @@ private void ContentGrid_ManipulationStarted(object sender, ManipulationStartedR
                 if (_leftCommandPanel != null)
                 {
                     _leftCommandTransform = _leftCommandPanel.RenderTransform as CompositeTransform;
+
+                    _leftCommandAnimation = new DoubleAnimation();
```

Comment:
Animation can be cancelled like this:
var anim = foo.offset(0, 5);

anim.Start();
...
anim.Stop()